native/src/components/MapView.tsx

```diff
@@ -46,14 +50,27 @@ const MapView = ({ boundingBox, featureCollection }: MapViewPropsType): ReactEle
     zoomLevel: defaultViewportConfig.zoom,
     bounds
   }
+
+  const updateLocationPermission = useCallback((locationPermission: boolean) => {
+    setLocationPermissionGranted(locationPermission)
+    setFollowUserLocation(false)
+    locationPermission && setFollowUserLocation(true)
```

Comment:
I never used && as an alernative for an if. I think this is simpler:

```suggestion
setFollowUserLocation(locationPermission)
```

Reply:
short circuit evaluation (&& or ||) is a common thing. i think for one liners not a bad code styling but it can be confusing in some cases.
in this case the question is why we should set the FollowUserLocation to false and then depending on locationPermission to true. setting the state two time should be avoided.

Reply:
Is removed now.